clang/test/CodeGen/debug-dir-win-pe-ltcg-string.c

```diff
@@ -0,0 +1,13 @@
+// This test checks if Window PE file compiled with -flto option contains a magic 
+// string "LTCG" to indicate LTO compilation.
+
+// REQUIRES: system-windows
```

Comment:
Why does this require Windows?

Reply:
Because of dumpbin; I will rewrite the tests to use objdump, see above